pkg/roachprod/failureinjection/failures/disk_stall.go

```diff
@@ -68,22 +71,46 @@ func (s *CGroupDiskStaller) Setup(ctx context.Context, l *logger.Logger, args Fa
 	// existing logs directory and copy the contents over after. If a symlink
```

Comment:
Maybe we could explain why this dance is necessary, as it's not immediately clear without looking at the implementation. We have to remind the reader that `{store-dir}/logs` is usually mounted on a different device than `logs`. Thus, `cp` is required. Since we can't rely on `mv`, (and even if we could, multiple FS operations aren't atomic), we need to stop the process.